GET private recipes

BNechita11 opened the pull request:
I created an endpoint that was supposed to retrieve a list of the user’s private recipes.

BNechita11 committed:
GET private recipes

en-madalina-marales commented on recipe-box-backend/src/main/java/com/endava/recipebox/service/impl/RecipeServiceImpl.java:
```diff
@@ -66,6 +71,14 @@ public List<RecipeDTO> getAllPublicRecipesByType(MealType mealType) {
 
+    @Override
+    public List<RecipeDTO> getAllPrivateRecipesByUserId(Long userId) {
+        return recipeMapper.map(recipeRepository.findAll().stream()
```
What will be the behavior in case there is no recipe found for userId?

BNechita11 committed:
Log message for missing private recipes

If no recipe is found for the given userId, the method will log a message saying "No private recipes found for userId: [userId]," and return an empty list.

BNechita11 committed:
Merge branch 'main' into feature/RCB-7-Get-Private-Recipes

BNechita11 committed:
Merge branch 'main' into feature/RCB-7-Get-Private-Recipes
